### DISCUSSION TITLE
For döngüsünde süslü parantez kullanımı

### DISCUSSION THREAD

**Discussion by B3Y0N4** [19/classroom#57] [C]
for döngüsünü kullanırken neden süslü {} parantezler  kullandığımızda döngü bir defa çalışır hiç kullanmadığımızda da normal çalişir 
    Bu kod farklı sonuc verirken 
    for(i = 0; i < n; i++)
        printf("%d ", *(ptr+i));
        return 0;
    Parantezleri eklediğmizde farklı sonuc verir 
    for(i = 0; i < n; i++){
        printf("%d ", *(ptr+i));
        return 0;
    }

**Comment by roktas** [comment]
Sorunu senin görmeni bekliyorum. :-)  Parantezli durumda tam olarak hangi satırlar çalışıyor ve ne oluyor?

**Reply by B3Y0N4** [reply]
anladım returnu 0; yanliş yerde yazmişim

**Reply by roktas** [reply]
Buradan bir ders çıkaralım. Bir sorun veya her sorun olduğunda hemen klavyeye sarılmayalım. Biraz daha düşünelim, çok düşünelim.

**Reply by B3Y0N4** [reply]
tamamdir hocam
